src/router/routes.js

```diff
@@ -184,6 +184,19 @@ export default [
 		],
 
 	},
+	{
+		path: '/settings/security',
+		component: () => import('@/pages/Settings/SecuritySettings'),
+		children: [{
+			path: '',
+			name: 'password',
```

Comment:
Are we sure about the name on this route? We don't typically have names on routes...

Reply:
removed, thanks for calling it out